apps/core/backup.py

```
@@ -83,11 +995,34 @@ def import_backup_xml(xml_content: bytes, dry_run: bool = False) -> Dict[str, An
+        "production_orders_created": 0,
+        "production_orders_updated": 0,
+        "portion_variants_created": 0,
+        "ingredient_overrides_created": 0,
+        "goods_receipts_created": 0,
+        "stock_transfers_created": 0,
+        "inventory_verifications_created": 0,
+        "stock_write_offs_created": 0,
     }
-
+    
     root = ET.fromstring(xml_content)
```

Ruff (0.14.14)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)